ansible-scylla-monitoring/defaults/main.yml

```diff
@@ -20,6 +20,7 @@ scylla_manager_ip: '127.0.0.1'
 # Download the scylla-monitoring from here
 scylla_monitoring_archive_url: 'https://github.com/scylladb/scylla-monitoring/archive/scylla-monitoring-3.10.1.tar.gz'
 
+scylla_monitoring_role_config_path: "{{ inventory_dir }}"
```

Comment:
Please, put a description for a new parameter

Reply:
Done